'Gap [mm]', 'Float', 20.0, '', 1.0]: numeric value 0 out of range

https://www.sirepo.com/radia#/copy-session/IS2Yc0Lm/visualization

```txt
Jun sirepo/uri_router.py:90:call_api api=<function api_copyNonSessionSimulation at 0x7fa046e11b70> exception=['Gap [mm]', 'Float', 20.0, '', 1.0]: numeric value 0 out of range stack=Traceback (most recent call last):
17:23:31 sirepo/uri_router.py", line 82, in call_api
17:23:31 r = flask.make_response(f(**kwargs) if kwargs else f())
17:23:31 sirepo/server.py", line 76, in api_copyNonSessionSimulation
17:23:31 res = _save_new_and_reply(data)
17:23:31 sirepo/server.py", line 707, in _save_new_and_reply
17:23:31 data = simulation_db.save_new_simulation(*args)
17:23:31 sirepo/simulation_db.py", line 538, in save_new_simulation
17:23:31 return save_simulation_json(data, do_validate=do_validate, uid=<redact>)
17:23:31 sirepo/simulation_db.py", line 575, in save_simulation_json
17:23:31 srschema.validate_fields(data, get_schema(data.simulationType))
17:23:31 sirepo/srschema.py", line 98, in validate_fields
17:23:31 _validate_number(val, sch_field_info)
17:23:31 sirepo/srschema.py", line 237, in _validate_number
17:23:31 raise AssertionError(util.err(sch_field_info, 'numeric value {} out of range', val))
17:23:31 AssertionError: ['Gap [mm]', 'Float', 20.0, '', 1.0]: numeric value 0 out of range
17:23:31 Exception was printed at:
17:23:31 AssertionError: ['Gap [mm]', 'Float', 20.0, '', 1.0]: numeric value 0 out of range
17:23:31 flask/app.py", line 2088, in __call__
17:23:31 return self.wsgi_app(environ, start_response)
17:23:31 flask/app.py", line 2070, in wsgi_app
17:23:31 response = self.full_dispatch_request()
17:23:31 flask/app.py", line 1513, in full_dispatch_request
17:23:31 rv = self.dispatch_request()
17:23:31 flask/app.py", line 1499, in dispatch_request
17:23:31 return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
17:23:31 sirepo/uri_router.py", line 222, in _dispatch
17:23:31 return call_api(route.func, kwargs)
17:23:31 sirepo/http_reply.py:284:_gen_exception_error unsupported exception=<class 'AssertionError'> msg=['Gap [mm]', 'Float', 20.0, '', 1.0]: numeric value 0 out of range
```

--- a/sirepo/package_data/static/json/radia-schema.json
+++ b/sirepo/package_data/static/json/radia-schema.json
@@ -570,7 +570,7 @@
         "undulator": {
             "_super": ["_", "model", "radiaObject"],
             "enableKickMaps": ["Enable Kick Maps", "Boolean", "1"],
-            "gap": ["Gap [mm]", "Float", 20.0, "", 1.0],
+            "gap": ["Gap [mm]", "Float", 20.0, "", 0.0],
             "gapAxis": ["Gap Axis", "BeamAxis", "y", "Direction of the magnetic gap - must differ from the beam axis"],
             "numPeriods": ["Number of Periods", "Integer", 2, "", 1],
             "periodLength": ["Period Length [mm]", "Float", 46.0],

